### PR TITLE
updated the URL and HTML tags for scrapping yahoo finance

### PR DESCRIPTION
**Pull request by Suman2023**
#### The existing Yahoo URL for scrapping the stock price is no longer maintained from August 26th, 2021 (attached the screenshot below). Moreover there is a change in HTML tags for the current price field. I have added the new URL and have updated the HTML tags required to fetch the current price of the stock.
Screenshot 2023-08-10 203953

- [ ] Add an algorithm?
- [x] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
- [x] All new Python files are placed inside an existing directory.
- [x] All filenames are in all lowercase characters with no spaces or dashes.
- [x] All functions and variable names follow Python naming conventions.
- [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
- [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
- [ ] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
- [x] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

**Comment by Suman2023** [comment]
This is my first contribution will really appreciate some feedback

File: web_programming/current_stock_price.py [hunk 1]
```diff
@@ -3,12 +3,20 @@
...
+    yahoo_finance_source = requests.get(url, headers={"USER-AGENT": "Mozilla/5.0"}).text
+    soup = BeautifulSoup(yahoo_finance_source, "html.parser")
+    specific_fin_streamer_tag = soup.find("fin-streamer", {"data-test": "qsp-price"})
 
+    if specific_fin_streamer_tag:
+        text = specific_fin_streamer_tag.get_text()
+        return text
+    else:
+        print("No <fin-streamer> tag with the specified data-test attribute found.")
+    return "Not Found"
```
Comment (tianyizheng02) [inline web_programming/current_stock_price.py:16]:
```suggestion
    if specific_fin_streamer_tag:
        text = specific_fin_streamer_tag.get_text()
        return text
    
    print("No <fin-streamer> tag with the specified data-test attribute found.")
    return "Not Found"
```

Also, why both print and return? Why not just return the error message? Or print the error message and return None?

**Review by tianyizheng02 — CHANGES_REQUESTED** [review]
LGTM for the most part, but I'll still have to verify whether the new URL works correctly myself when I get the chance.

File: web_programming/current_stock_price.py (hunk 1 above)
Comment (Suman2023) [reply web_programming/current_stock_price.py:16]: Thanks for the suggestion. Cannot return None as that would through an error in string formatting. But returning the error looks valid. Doing the change for that.

**Review by tianyizheng02 — APPROVED** [review]
Verified that the fix works, LGTM

**Pull request merged**